src/Events.Functions/Services/Interfaces/IWebhookService.cs

```diff
@@ -1,4 +1,6 @@
+using System.Net.Http;
```

Comment:
New using without any code changes that could be using anything from the namespace.